src/global/Constants.h

```diff
@@ -60,6 +60,8 @@ static constexpr std::pair<std::string_view, std::string_view> GEOF_PREFIX = {
     "geof:", "http://www.opengis.net/def/function/geosparql/"};
 static constexpr std::pair<std::string_view, std::string_view> MATH_PREFIX = {
     "math:", "http://www.w3.org/2005/xpath-functions/math#"};
+static constexpr std::pair<std::string_view, std::string_view> SCHEMA_TYPE = {
+    "xml_schema", "http://www.w3.org/2001/XMLSchema#"};
```

Comment:
```suggestion
static constexpr std::pair<std::string_view, std::string_view> XSD_PREFIX= {
    "xsd:", "http://www.w3.org/2001/XMLSchema#"};
```
(That is the "canonical" name for this prefix).